azure-pipelines.yml

```diff
@@ -57,16 +57,36 @@ jobs:
       toolMajorVersion: V2
       suppressionsFile: './scripts/ci/credscan/CredScanSuppressions.json'
       toolVersionV2: '2.1.17'
+
   - task: ms-codeanalysis.vss-microsoft-security-code-analysis-devops.build-task-postanalysis.PostAnalysis@1
     displayName: 'Post Analysis'
     inputs:
       AllTools: false
       BinSkim: false
       CredScan: true
+      PoliCheck: false
       RoslynAnalyzers: false
       TSLint: false
       ToolLogsNotFoundAction: 'Standard'
 
+- job: PolicyCheck
+  displayName: "Policy Check"
+  pool:
+    name: ${{ variables.windows_pool }}
+  steps:
+  - task: securedevelopmentteam.vss-secure-development-tools.build-task-policheck.PoliCheck@2
+    displayName: 'Run Policy Check'
+    inputs:
+      targetType: F
+      result: PoliCheckResult.xml
+
+  - task: securedevelopmentteam.vss-secure-development-tools.build-task-postanalysis.PostAnalysis@2
+    displayName: 'Post Analysis'
+    inputs:
+      GdnBreakAllTools: false
```

Comment:
Gdn means [guardian](https://eng.ms/docs/security-compliance-identity-and-management-scim/security/azure-security/cloudai-security-fundamentals-engineering/security-integration/guardian-wiki/microsoft-guardian/guardian-commands/break).

Reply:
All documents can be easily found by clicking the button on the UI
![image](https://user-images.githubusercontent.com/18628534/202380960-f638ce88-497d-42d6-a4e4-eb1472926bd9.png)